### DISCUSSION TITLE
Convert a subtask to an issue is not working for me.

### DISCUSSION THREAD

**Discussion by vvs-basanth** [freeCodeCamp/Developer_Quiz_Site#1061] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/46da5815-8cdb-45e8-9a86-6c99f84140e4)

[Issue assigned](
https://github.com/freeCodeCamp/Developer_Quiz_Site/issues/1055)

Clicking "Convert to issue" is not creating any new issues. I'm getting the `We are unable to convert the task to an issue at this time. Please try again.` this kind of toast notification.

I can't tell if its a permissions issue or an outage atm.
I searched up online but could not find anything which could solve this.

**Comment by jdwilkin4** [comment, marked as answer]
Yeah, it is a permissions issue. Probably because you are not a fcc member.

I was able to create an issue from a task just fine.

https://github.com/freeCodeCamp/Developer_Quiz_Site/issues/1062

So you can just manually copy some of those tasks over and manually create an issue.
Because not every single task you listed needs its own issue anyway. A lot of those cane be combined where applicable. 

As long as this EPIC is broken up into smaller issues so it is easier to review and test that is all that matters
The point is to not create one giant PR that will be a nightmare to review 👍

**Reply by vvs-basanth** [reply]
Ohh okay
Thanx..
